backend/src/main/java/org/sefglobal/core/repository/AmbassadorRepository.java

```diff
@@ -10,4 +12,5 @@
 public interface AmbassadorRepository extends JpaRepository<Ambassador, Long> {
     Ambassador findByIdAndStatus(long id, String status);
     List<Ambassador> findAllByStatus(String status);
+    Page<Ambassador> findAll(Pageable pageable);
```

Comment:
No need to create a new method `Page<Ambassador> findAll(Pageable pageable)` there's one already existing in the parent repository classes. Use that.

Reply:
I will use it.